packages/gatsby-source-wordpress/src/gatsby-node.js

```diff
@@ -70,6 +71,7 @@ exports.sourceNodes = async (
   _includedRoutes = includedRoutes
   _excludedRoutes = excludedRoutes
   _keepMediaSizes = keepMediaSizes
+  _restApiRoutePrefix = restApiRoutePrefix
```

Comment:
`_restApiRoutePrefix` is undefined, it needs to be added above where the other _vars are declared.

Reply:
Done! :)

Reply:
Awesome, thanks!